<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>run1s</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Org</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comments</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>376</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionEnterprise"/>
    <addaction name="actionEnterprise_singleuser"/>
    <addaction name="actionConfigurer"/>
    <addaction name="actionMonitor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="FileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="OptionsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QToolBar" name="ActionsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnterprise"/>
   <addaction name="actionEnterprise_singleuser"/>
   <addaction name="actionConfigurer"/>
   <addaction name="actionMonitor"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/exit_32x32.png</normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/exit_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/qt_32x32.png</normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/qt_32x32.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/settings_32x32.png</normaloff>../../../../../../../../../home/eugene/Projects/uprojects/run1s/src/client/cpp/run1s/icons/settings_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEnterprise">
   <property name="icon">
    <iconset>
     <normaloff>icons/ent_16x16.png</normaloff>icons/ent_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Enterprise</string>
   </property>
  </action>
  <action name="actionEnterprise_singleuser">
   <property name="icon">
    <iconset>
     <normaloff>icons/ent_16x16.png</normaloff>icons/ent_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Enterprise (singleuser)</string>
   </property>
  </action>
  <action name="actionConfigurer">
   <property name="icon">
    <iconset>
     <normaloff>icons/cfg_16x16.png</normaloff>icons/cfg_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Configurer</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="icon">
    <iconset>
     <normaloff>icons/mon_16x16.png</normaloff>icons/mon_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
